src/main/features/statement-of-means/paths.ts

```diff
@@ -0,0 +1,8 @@
+import { RoutablePath } from 'common/router/routablePath'
+
+const statementOfMeansPath = '/case/:externalId/statement-of-means'
+
+export class Paths {
+  static readonly startPage = new RoutablePath(`${statementOfMeansPath}/start`)
```

Comment:
shouldn't this be under the response journey?

Reply:
Yeah, probably makes sense to add it there.

Reply:
Doesn't work too nicely with features though :(